autodeploy.js

```diff
@@ -16,10 +16,10 @@ function parseToken(token) {
 }
 
 class AutoDeployApi {
-  constructor(url, token) {
+  constructor(url, token, timeout) {
     this.url = url.replace(/\/*$/, "");
     this.client = new HttpClient("autodeploy-action", [parseToken(token)], {
-      socketTimeout: 6 * 60000,
+      socketTimeout: timeout,
```

Comment:
The timeout used to be `6 * 60_000ms = 360_000ms = 6 min`.
After this PR it's `60_000 = 1 min`.
I believe this is unintended?

Reply:
6 minutes as a default timeout seemed to long to me (it's quite uncommon for http requests to have such a long timeout). Less pod-heavy infras should be able to restart in less than a minute. I should have mentioned this in the PR description though (will add)